src/exercise/01.js

```diff
@@ -3,27 +3,25 @@
 
 import * as React from 'react'
 
-function Greeting() {
-  // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+function Greeting({initialName = ''}) {
+  const [name,setName] = React.useState(initialName)
```

Comment:
```suggestion
  const [name, setName] = React.useState(props.initialName);
```

Reply:
yeah see i did that first but then his version didn't have ittttttttt ah well.